src/main/java/net/minecraftforge/common/DimensionManager.java

```diff
@@ -412,42 +411,37 @@ public static void unloadWorlds(Hashtable<Integer, long[]> worldTickTimes)
      * block of free ids. Always call for each individual ID you wish to get.
      * @return the next free dimension ID
      */
-    public static int getNextFreeDimId() {
+    public static int getNextFreeDimId()
+    {
         int next = 0;
         while (true)
```

Comment:
Maybe
```suggestion
        while (usedIds.contains(next) || !checkAvailable(next)) {
            next++;
        }
```
I dislike "infinite" loops ;)